executor/executor_write.go

```diff
@@ -438,6 +440,40 @@ func (e *LoadDataInfo) InsertData(prevData, curData []byte) ([]byte, error) {
 	return curData, nil
 }
 
+func escapeCols(strs [][]byte) []string {
+	ret := make([]string, len(strs))
+	for i, v := range strs {
+		ret[i] = string(escape(v))
+	}
+	return ret
+}
+
+func escape(str []byte) []byte {
+	pos := 0
+	for i := 0; i < len(str); i++ {
+		c := str[i]
+		if c == '\\' {
+			if i+1 < len(str) {
+				switch str[i+1] {
+				case 't':
```

Comment:
There are many escape chars in MySQL. Should we handle them? Can we use the bytes escape function in scanner?